salt/kubelet/stop.sls

```diff
@@ -4,6 +4,14 @@
 include:
   - kubectl-config
 
+{% if not "kube-master" in salt['grains.get']('roles', []) %}
+set-should-uncordon-grain:
+  grains.present:
+    - name: kubelet:should_uncordon
+    - value: {{ salt['cmd.run']("kubectl --kubeconfig=" + pillar['paths']['kubeconfig'] + " get nodes " + grains['caasp_fqdn'] + " -o=jsonpath='{.spec.unschedulable}' 2>/dev/null") != "true" }}
+    - force: True
+{% endif %}
```

Comment:
See [this comment](https://github.com/kubic-project/salt/pull/339/files#r158726435)